cgi-bin/bml/scheme/global.look

```diff
@@ -13,165 +13,25 @@
 
```

Comment:
Oh wow global.look it's been a _hot minute_ since I've peered behind these particular curtains.